### ISSUE TITLE
Issue while updating schema (atoti 0.7.1)

### ISSUE THREAD

**Issue by Wi1iku** [atoti/atoti#722]
Hi, I've encountered a similar bug to #363 and #460 
## Steps to reproduce
`m["CurrienciesClose"]=tt.agg.mean(df_currencies["Close"])`
`m["CurrienciesOpen"]=tt.agg.mean(df_currencies["Open"])`
`m["CurrienciesHigh"]=tt.agg.mean(df_currencies["High"])`
`m["CurrienciesLow"]=tt.agg.mean(df_currencies["Low"])`

Drive with the notebook and data: https://drive.google.com/file/d/1yrjoxWtqt0xmHT6wzhwr_wsIdlGXMtbh/view?usp=share_link

![image](https://user-images.githubusercontent.com/77934237/201040470-9db55fd4-629f-4259-b289-08ed68eec350.png)
After reloading the cell, the session creates the measures as expected

After re-executing, works as expected

![image](https://user-images.githubusercontent.com/77934237/201049097-36ed16ac-5af4-4ce7-9a8e-dfb851bf77a0.png)

## Actual Result

AtotiJavaException: [500] Issue while updating schema: Trading
After reloading the cell, the measure is created

## Expected Result

Measures created in the first try

## Environment

- atoti: 0.7.1
- Python: 3.10.4
- Operating system: win32

## Logs (if relevant)

logs too long, txt with logs: 
[issuelogs.txt](https://github.com/atoti/atoti/files/9979257/issuelogs.txt)

**Comment by mailys-ds** [comment]
Hi, thanks for reporting this issue, I can reproduce it with your notebook, we'll work on a fix.

**Comment by mailys-ds** [comment]
Hi,
This will be fixed in atoti 0.7.2.
As a workaround before the release, you can create a hierarchy from your table column.
``` python
h["Close"] = [df_currencies["Close"]]
m["CurrienciesClose"]=tt.agg.mean(df_currencies["Close"])
```
